packages/components/psammead-navigation/src/DropdownNavigation/index.jsx

```diff
@@ -0,0 +1,232 @@
+import React from 'react';
+import styled, { css } from 'styled-components';
+import { shape, string, node, bool, func, oneOf } from 'prop-types';
+import VisuallyHiddenText from '@bbc/psammead-visually-hidden-text';
+import { navigationIcons } from '@bbc/psammead-assets/svgs';
+import {
+  C_WHITE,
+  C_POSTBOX,
+  C_EBON,
+  C_SHADOW,
+} from '@bbc/psammead-styles/colours';
+import {
+  GEL_SPACING_HLF,
+  GEL_SPACING,
+  GEL_SPACING_DBL,
+} from '@bbc/gel-foundations/spacings';
+import { GEL_GROUP_3_SCREEN_WIDTH_MIN } from '@bbc/gel-foundations/breakpoints';
+import { getPica } from '@bbc/gel-foundations/typography';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+import { getSansRegular } from '@bbc/psammead-styles/font-styles';
+
+const MENU_ICON_SIDE_LENGTH = '2.75rem'; // 44px
+
+const getStyles = dir => {
+  const direction = dir === 'ltr' ? 'left' : 'right';
+  return `border-${direction}: ${GEL_SPACING_HLF} solid ${C_WHITE};
+          padding-${direction}: ${GEL_SPACING};`;
+};
+
+const DropdownWrapper = styled.div`
+  background-color: ${C_EBON};
+
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
+    display: none;
```

Comment:
and visibility hidden please?

Reply:
I'm not 100% sure about this convention - do we know of browsers or situations that don't support `display: none`? And if they don't would we not need to deal with the fact the element still affects the layout?

Reply:
I can't actually find any reference to this on the net, this is something we have used for years, compatibility may of improved since we started using it (sorry haven't had time to research), considering the number of AT combo's and versions that could potentially be used across the world I certainly don't think it harms though